README.md

```diff
@@ -122,13 +145,20 @@ e.g.
     "title_number": "DN100",
     "borrowers": [
         {
-            "forename": "fred",
-            "middle_name": "joe",
-            "surname": "bloggs"
+            "forename": "lisa",
+            "middle_name": "ann",
+            "surname": "bloggette",
+            "address": "example address with postcode, PL14 3JR",
+            "dob": "23/01/1986",
+            "phone_number": "07502154062"
         },
         {
-            "forename": "lisa",
-            "surname": "bloggette"
+            "forename": "frank",
+            "middle_name": "ann",
+            "surname": "bloggette",
+            "address": "Example address with postcode, PL14 3JR",
+            "dob": "23/01/1986",
```

Comment:
Since this is going to be an API used by customers it might be worth thinking about conforming to a standard for date, if you dont conform to a standard then the user could be wondering "is it dd/mm/yyyy or mm/dd/yyyy" here is the iso 8601 that I thing is generally best to use https://en.wikipedia.org/wiki/ISO_8601